src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -60,9 +60,17 @@ Click **Next: Capacity**.
 
 ## Step 4. Enable VPC Peering (optional)
 
-You can use [VPC peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering) to connect a GCP application to a CockroachDB {{ site.data.products.cloud }} cluster deployed on GCP. A separate VPC Peering connection is required for each cluster.
+You can use [VPC peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering) to establish a private connection between a GCP application and a CockroachDB {{ site.data.products.dedicated }} cluster deployed on GCP. A separate VPC Peering connection is required for each cluster.
 
-VPC peering is only available for GCP clusters. For clusters deployed on AWS, you can [set up AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster. [Azure Virtual Network Peering](https://learn.microsoft.com/azure/virtual-network/virtual-network-peering-overview) is not yet supported. Refer to [CockroachDB {{ site.data.products.dedicated }} on Azure]({% link cockroachcloud/cockroachdb-dedicated-on-azure.md %}).
+{{site.data.alerts.callout_success}}
+You can [set up Private Service Connect (Preview)]({% link cockroachcloud/network-authorization.md %}#private-service-connect) instead of VPC peering after creating your cluster.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For clusters deployed on AWS, you can [set up AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster.
+{{site.data.alerts.end}}
+
+[Azure Virtual Network Peering](https://learn.microsoft.com/azure/virtual-network/virtual-network-peering-overview) is not yet supported. Refer to [CockroachDB {{ site.data.products.dedicated }} on Azure]({% link cockroachcloud/cockroachdb-dedicated-on-azure.md %}).
 
 You can use CockroachDB {{ site.data.products.cloud }}'s default IP range and size (`172.28.0.0/14`) as long as it doesn't overlap with the IP ranges in your network. Alternatively, you can configure the IP range:
 
```

Comment:
Reword Step 3 to include GCP Private Service Connect. Currently, it only has VPC Peering or AWS PrivateLink
<img width="1360" alt="image" src="https://github.com/cockroachdb/docs/assets/11891727/1482e516-ef23-462a-b04e-08715e8b7b49">